Jenkinsfile

```diff
@@ -33,34 +34,57 @@ pipeline {
                 setBuildStatus("Preparing", "PENDING")
             }
         }
-
         stage('Get okta.properties') {
-                    agent {
-                        node {
-                            label 'dev_mesos_v2'
-                            customWorkspace "jenkins-mtsdk-${currentBuild.number}"
-                        }
-                    }
-                    steps {
-                        script {
-                            def pipelineLibrary = library('pipeline-library').com.genesys.jenkins
-                            def testing = pipelineLibrary.Testing.new()
-                            oktaproperties = testing.getSecretStashSecret(
-                                'dev',
-                                'us-east-1',
-                                'mobiledx-ios',
-                                'okta-properties',
-                                env.WORKSPACE
-                            )
-                            echo "okta.properties fetched successfully."
-                        }
-                    }
+            agent {
+                node {
+                    label 'dev_mesos_v2'
+                    customWorkspace "jenkins-mtsdk-${currentBuild.number}"
+                }
+            }
+            steps {
+                script {
+                    def pipelineLibrary = library('pipeline-library').com.genesys.jenkins
```

Comment:
Looks like we are obtaining the reference to pipelinLibrary twice. Once for oktaProperties, second time for google-services. 
Is it possible to somehow combine them into single step, like "retrieve values from secret stash" or something ?